webpack/set-webpack-public-path-plugin/README.md

```diff
@@ -0,0 +1,133 @@
+`https://mycdn.net/files/build_id/assets/`.
+
+This option is exclusive to other options. If it is set, `systemJs`, `publicPath`, and `urlPrefix` will be ignored.
+
+#### `systemJs = true`
+
+Use `System.baseURL` if it is defined.
+
+#### `publicPath = '...'`
+
+Use the specified path as the base public path. If `urlPrefix` is also defined, the public path will
+be the concatenation of the two (i.e. - `__webpack_public_path__ = URL.concat({publicPath} + {urlPrefix}`).
+This option takes precedence over the `systemJs` option.
+
+#### `urlPrefix = '...'`
+
+Use the specified string as a URL prefix after the SystemJS path or the `publicPath` option. If neither
+`systemJs` nor `publicPath` is defined, this option will not apply and an exception will be thrown.
+
+#### `regexVariable = '...'`
+
+Check for a variable with name `...` on the page and use its value as a regular expression against script paths to
+the bundle's script. If a value `foo` is passed into `regexVariable`, the produced bundle will look for a variable
+called `foo` during initialization, and if a `foo` variable is found, use its value as a regular expression to
+detect the bundle's script.
+
+For example, if the `regexVariable` option is set to `scriptRegex` and `scriptName` is set to `{ name: 'myscript' }`,
+consider two cases:
+
+##### Case 1
+
+```HTML
+<html>
+  <head>
+    <script>
+      var scriptRegex = /thescript/i;
+    </script>
+    <script src="theScript.js"></script>
+  </head>
+
+  ...
+</html>
+```
+
+In this case, because there is a `scriptRegex` variable defined on the page, the bundle will use its value
+(`/thescript/i`) to find the script.
+
+##### Case 2
+
+```HTML
+<html>
+  <head>
+    <script src="myScript.js"></script>
+  </head>
+
+  ...
+</html>
+```
+
+In this case, because there is not a `scriptRegex` variable defined on the page, the bundle will use the value
+passed into the `scriptName` option to find the script.
+
+# SystemJS Caveat
+
+When modules are loaded with SystemJS (and with the , `scriptLoad: true` meta option) `<script src="..."></script>`
+tags are injected onto the page, evaludated and then immediately removed. This causes an issue because they are removed
+before webpack module code begins to execute, so the `publicPath=...` option won't work for modules loaded with SystemJS.
+
+To circumvent this issue, a small bit of code is availble to that will maintain a global register of script paths
+that have been inserted onto the page. This code block should be appended to bundles that are expected to be loaded
+with SystemJS and use the `publicPath=...` option.
+
+## `getGlobalRegisterCode(bool)`
+
+This function returns a block of JavaScript that maintains a global register of script tags. If the optional boolean paramter
+is set to `true`, the code is not minified. By default, it is minified.
+
+## Usage
+
+``` javascript
+var setWebpackPublicPath = require('@microsoft/loader-set-webpack-public-path');
```

Comment:
It's not, I just missed a few things in the updates.

---
In reply to: [130456600](https://github.com/Microsoft/web-build-tools/pull/288#discussion_r130456600) [](ancestors = 130456600)